mybatis-spring-boot-autoconfigure/pom.xml

```diff
@@ -99,24 +103,19 @@
       <artifactId>h2</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-jdbc</artifactId>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.flywaydb</groupId>
-      <artifactId>flyway-core</artifactId>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-flyway</artifactId>
```

Comment:
```suggestion
<artifactId>spring-boot-flyway</artifactId>
```

Reply:
@kazuki43zoo In Spring Boot 4.0, the Spring team optimized the component modules by only adding dependencies for related modules .